src/core_plugins/kibana/public/dashboard/panel/panel_header.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { KuiKeyboardAccessible } from 'ui_framework/components';
+import { PanelOptionsMenu } from './panel_options_menu';
+
+export class PanelHeader extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getOptionsDropDown() {
+    return (
+      <PanelOptionsMenu
+        onEditPanel={this.props.onEditPanel}
+        onDeletePanel={this.props.onDeletePanel}
+        onToggleExpandPanel={this.props.onToggleExpand}
+        isExpanded={this.props.isExpanded}
+      />
+    );
+  }
+
+  getExpandToggle() {
+    const { isExpanded } = this.props;
+    const classes = classNames('kuiIcon', null, { 'fa-expand': !isExpanded, 'fa-compress': isExpanded });
+    return (
+      <KuiKeyboardAccessible>
+        <a
+          className="kuiMicroButton"
+          aria-label="Expand panel"
+          data-test-subj="dashboardPanelExpandIcon"
+          onClick={this.props.onToggleExpand}
```

Comment:
Could you add an `aria-expanded` which has either `true` or `false` as value, depending whether the popup is currently expanded or not.

Please also add an `aria-haspopup="true"` to the element to indicate it has a popup that can be shown.

Please also add an `aria-controls={someId}` to the element, with the `id` of the element, that will be expanded and collapsed.  You can use the [htmlIdGenerator](https://github.com/elastic/kibana/blob/master/style_guides/accessibility_guide.md#how-to-generate-ids) from the `ui_framework` to generate those `id`s.

Reply:
I see now my word choice was poor.  This toggle isn't associated with a pop up.  I should perhaps rename to toggleMinMax or something, but since these semantics already existed prior to this PR, I'd rather not do it in here.